main.py

```diff
@@ -0,0 +1,162 @@
+import csv
+import json
+import datetime
+import constant
+import pandas as pd
+from youtubesearchpython import *
+from telebot import TeleBot, types, ContinueHandling
+from telebot.callback_data import CallbackData
+from CallBackFilter import CallBackFilter
+from dbAlchemy import DbSqlAlchemy
+
+db = DbSqlAlchemy()
+db.setup()
+
+
+def csv_to_list(file_csv):
+    return list(csv.DictReader(open(file_csv, 'r')))
+
+
+def song_titles(file_csv):
+    df = pd.read_csv(file_csv)
+    return '\n'.join('\U00002b50 ' + str(e) for e in list(df["track"]))
+
+
+KPOP = csv_to_list('Data/kpop.csv')
+ROCK = csv_to_list('Data/rock.csv')
+bts = song_titles('Data/bts.csv')
+exo = song_titles('Data/exo.csv')
+elton = song_titles('Data/elton.csv')
+queen = song_titles('Data/queen.csv')
```

Comment:
When there will be a lot of records which not going to fit into your memory RAM. 
Your script is going to crash. 
User database for reading data from the disk instead of using CSV files. 
Storing them inside in CSV file is not extensible.

Reply:
I have fixed it by not loading the whole files but only two random elements from the file